okhttp/src/main/kotlin/okhttp3/Cookie.kt

```diff
@@ -262,6 +264,20 @@ class Cookie private constructor(
     private var persistent = false
     private var hostOnly = false
 
+    constructor()
+
+    internal constructor(cookie: Cookie) {
```

Comment:
Can this be private?

Reply:
No, Cookie can't access Cookie.Builder(Cookie) if it is.